pandas/_libs/tslib.pyx

```diff
@@ -617,6 +647,8 @@ cpdef array_to_datetime(ndarray[object] values, str errors='raise',
                     # A ValueError at this point is a _parsing_ error
                     # specifically _not_ OutOfBoundsDatetime
                     if _parse_today_now(val, &iresult[i]):
+                        # TODO: Do we treat this as local?
```

Comment:
I feel like we should follow `datetime.now` and `datetime.today` conventions and return local for both. IIRC there was discussion somewhere around this issue?

Reply:
I do think that `to_datetime("now")` should match `Timestamp("now")`. I am not sure if I fully understand your example's point.

Reply:
> I am not sure if I fully understand your example's point.

The point is that `to_datetime([a, b, c])[1:]` should not depend on `a` (except possibly for whether it is object-dtype)

Reply:
@mroeschke thoughts on this?